packages/esm-patient-chart-app/src/visit/visit-form/visit-form.component.tsx

```diff
@@ -203,9 +206,8 @@ const StartVisitForm: React.FC<StartVisitFormProps> = ({
       visitStartDate,
       visitStartTime: dayjs(visitStartDate).format('hh:mm'),
       visitStartTimeFormat: visitStartDate.getHours() >= 12 ? 'PM' : 'AM',
-
-      visitType: visitToEdit?.visitType?.uuid,
-      visitLocation: visitToEdit?.location ?? sessionLocation ?? {},
+      visitType: visitToEdit?.visitType?.uuid ?? emrConfiguration?.atFacilityVisitType?.uuid,
```

Comment:
Set the visit type to the "atFacilityVisitType" as defined by the EMR API module, if present.